docs/RENEW_CI_CREDENTIALS.md

```
@@ -0,0 +1,170 @@
+authentication / expiration error.
+
+---
+
+## Snap Store (Linux)
+
+Used by `snapcore/action-publish@v1` step in
+`.github/workflows/pull-request-build.yml` and `build-release.yml`.
+
+**Symptom in CI log:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)